easybuild/framework/easyblock.py

```diff
@@ -3045,7 +3053,15 @@ def inject_checksums(ecs, checksum_type):
                         else:
                             exts_list_lines.append("%s'checksums': [" % (INDENT_4SPACES * 2))
                             for fn, checksum in ext_checksums:
-                                exts_list_lines.append("%s'%s',  # %s" % (INDENT_4SPACES * 3, checksum, fn))
+                                line_indent = INDENT_4SPACES * 3
+                                ext_checksum_line = "%s'%s',  # %s" % (line_indent, checksum, fn)
+                                if len(ext_checksum_line) > MAX_LINE_LENGTH:
+                                    exts_list_lines.extend([
+                                        "%s# %s" % (line_indent, fn),
+                                        "%s'%s'," % (line_indent, checksum),
+                                    ])
+                                else:
+                                    exts_list_lines.append(ext_checksum_line)
```

Comment:
@smoors This diff makes it really stand out that this is a whole bunch of duplicate code, which we should avoid...

We should look into adding an (inner?) function that takes care of this once, for a specified indent level, so we can use it in both situations...

Let me know if you're up for looking into that or not.

Reply:
ok, I will look into it.